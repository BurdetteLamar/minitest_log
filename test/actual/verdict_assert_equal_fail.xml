<log>
  <summary verdicts='3' failures='2' errors='0'/>
  <section name='Test' method='verdict_assert_equal?' type='fail'>
    <section name='Args=[0, 1]'>
      <verdict method='verdict_assert_equal?' outcome='failed' id='0' message='Message 0'>
        <exp_value>
          0
        </exp_value>
        <act_value>
          1
        </act_value>
        <exception class='Minitest::Assertion'>
          <message>
            Expected: 0 Actual: 1
          </message>
          <backtrace/>
        </exception>
      </verdict>
    </section>
    <section name='Args=[:a, :b]'>
      <verdict method='verdict_assert_equal?' outcome='failed' id='1' message='Message 1'>
        <exp_value>
          :a
        </exp_value>
        <act_value>
          :b
        </act_value>
        <exception class='Minitest::Assertion'>
          <message>
            Expected: :a Actual: :b
          </message>
          <backtrace/>
        </exception>
      </verdict>
    </section>
  </section>
  <verdict method='verdict_assert_equal?' outcome='passed' id='__error_count__'>
    <exp_value>
      0
    </exp_value>
    <act_value>
      0
    </act_value>
  </verdict>
</log>
