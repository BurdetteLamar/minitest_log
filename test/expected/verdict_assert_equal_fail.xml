<log>
  <summary_ verdicts='3' failures='2' errors='0'/>
  <section_ name='Test' method='verdict_assert_equal?' type='fail'>
    <section_ name='Args=[0, 1, &quot;Message 0&quot;]'>
      <verdict_ method='verdict_assert_equal?' outcome='failed' id='0' message='Message 0'>
        <expected_ class='Integer' value='0'/>
        <actual_ class='Integer' value='1'/>
        <exception_ class='Minitest::Assertion' message='Expected: 0\n  Actual: 1'>
          <backtrace_>
          </backtrace_>
        </exception_>
      </verdict_>
    </section_>
    <section_ name='Args=[:a, :b, &quot;Message 1&quot;]'>
      <verdict_ method='verdict_assert_equal?' outcome='failed' id='1' message='Message 1'>
        <expected_ class='Symbol' value=':a'/>
        <actual_ class='Symbol' value=':b'/>
        <exception_ class='Minitest::Assertion' message='Expected: :a\n  Actual: :b'>
          <backtrace_>
          </backtrace_>
        </exception_>
      </verdict_>
    </section_>
  </section_>
  <verdict_ method='verdict_assert_equal?' outcome='passed' id='error_count'>
    <expected_ class='Integer' value='0'/>
    <actual_ class='Integer' value='0'/>
  </verdict_>
</log>
